src/pages/dashboard/dashboard.html

```diff
@@ -61,7 +61,7 @@ <h1 id="hello" text-center>Hello, {{ this.name }}!</h1>
             <ion-row>
               <ion-col>
                 <h2 class="action-item-title">Current Action Item:</h2>
-                <p>Complete Pre-Separation Counseling</p>
+                <p id="currentActionItemBody">Complete Pre-Separation Counseling</p>
```

Comment:
We should try not to use ID's when possible. ID's tend to cause specificity issues down the line and can be hard to manage/debug for.